extras/fixture/src/unity_fixture.c

```diff
@@ -151,7 +159,7 @@ static int malloc_fail_countdown = MALLOC_DONT_FAIL;
 
 void UnityMalloc_StartTest(void)
 {
-    malloc_count = 0;
+    malloc_count          = 0;
```

Comment:
This is strange. I understand where you are filling a struct you would want to have `=` aligned, but here it is a separate variable...

Reply:
Actually there were both types in the code base if you look closely:
- https://github.com/ThrowTheSwitch/Unity/blob/c9b1d217fd0cb26929d3554da1377e90aa583198/src/unity.c#L1015-L1018
- https://github.com/ThrowTheSwitch/Unity/blob/c9b1d217fd0cb26929d3554da1377e90aa583198/src/unity.c#L1091-L1093

I decided to take the one that looked cleaner to me and since we aligned a lot of stuff in the code it also seemed more consistent to use this alignment here as well.